### PR TITLE
Map particle stuff

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

File: mappings/net/minecraft/client/particle/AbstractDustParticle.mapping [hunk 1]
```diff
@@ -8,4 +8,8 @@ CLASS net/minecraft/class_5736 net/minecraft/client/particle/AbstractDustParticl
 		ARG 8 velocityX
 		ARG 10 velocityY
 		ARG 12 velocityZ
+		ARG 14 parameters
 		ARG 15 spriteProvider
+	METHOD method_33076 addFuzziness (FF)F
```
Comment (liach) [inline mappings/net/minecraft/client/particle/AbstractDustParticle.mapping]: This is more like bleaching the color. The 2nd param is a universal bleach while this method applies extra less (but different) bleaches to each of the 3 color components

File: mappings/net/minecraft/client/particle/DustColorTransitionParticle.mapping [hunk 2]
```diff
@@ -7,7 +9,13 @@ CLASS net/minecraft/class_5734 net/minecraft/client/particle/DustColorTransition
 		ARG 8 velocityX
 		ARG 10 velocityY
 		ARG 12 velocityZ
+		ARG 14 parameters
 		ARG 15 spriteProvider
+	METHOD method_33073 addFuzziess (Lnet/minecraft/class_1160;F)Lnet/minecraft/class_1160;
```
Comment (liach) [inline mappings/net/minecraft/client/particle/DustColorTransitionParticle.mapping]: Same comment as 33076

File: mappings/net/minecraft/client/particle/VibrationParticle.mapping [hunk 3]
```diff
@@ -1,5 +1,13 @@
 CLASS net/minecraft/class_5737 net/minecraft/client/particle/VibrationParticle
 	FIELD field_28249 vibration Lnet/minecraft/class_5720;
+	METHOD <init> (Lnet/minecraft/class_638;Lnet/minecraft/class_5720;I)V
+		ARG 1 world
+		ARG 2 vibration
+		ARG 3 maxAge
+	METHOD method_33078 (Lnet/minecraft/class_4588;Lnet/minecraft/class_4184;FLjava/util/function/Consumer;)V
```
Comment (liach) [inline mappings/net/minecraft/client/particle/VibrationParticle.mapping]: This seems to be just drawing the vibration particle. In addition, the consumer can probably be named `transforms`

**Comment by liach** [comment]
Particle names are truly messy; some follow bad mojang registry names (`DustColorTransitionParticle`) while some don't follow at all (`FireworksSparkParticle`). Might find a day for normalizing these names.

File: mappings/net/minecraft/client/particle/AbstractDustParticle.mapping (hunk 1 above)
Comment (enbrain) [reply mappings/net/minecraft/client/particle/AbstractDustParticle.mapping]: It actually does the opposite of whitening.

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**